src/api/app/controllers/source_controller.rb

```diff
@@ -517,7 +517,8 @@ def update_project_meta
         # FIXME3.0: don't modify send data
         project.relationships.build(user: User.current, role: Role.find_by_title!('maintainer'))
       end
-      project.store
+      opts = { :comment => params[:comment] }
+      project.store opts
```

Comment:
I tried t it  without the ( ) and this did not work, with the () it runs, so I will send a new request.